app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -147,6 +147,10 @@ def associated_host(vm)
       associate_by("ip", [vm.public_ip_address, vm.private_ip_address])
     end
 
+    def associated_vm(host)
```

Comment:
This then needs to be done for all compute resources which are available as a plugin. Like foreman_fog_proxmox; foreman_azure_rm, ...

Reply:
Hey @sbernhard thanks for your input, and sorry i didn´t get back to it in all that time. But hopefully I'll get to it soon.

Reply:
@sbernhard I added your suggestion to the PR.